src/ast/semantic_analyser.h

```diff
@@ -65,6 +65,7 @@ class SemanticAnalyser : public Visitor {
   bool check_symbol(const Call &call, int arg_num);
 
   void check_stack_call(Call &call, Type type);
+  inline void Error(const std::string &msg, const location & loc);
```

Comment:
```suggestion
  void error(const std::string &msg, const location & loc);
```
Lowercase 'e' to match the rest of the functions
Correct me if I'm wrong, but I don't think 'inline' actually does anything useful here - the compiler will decide what to inline by itself